queryscript/src/compile/sql.rs

```diff
@@ -144,6 +144,14 @@ pub fn select_star_from(relation: sqlast::TableFactor) -> sqlast::Query {
     )
 }
 
+pub fn select_limit_0(mut query: sqlast::Query) -> sqlast::Query {
```

Comment:
Would it be annoying to do this without mutating the argument?  If so, that's fine, I'd just request we change the name to something like `set_limit_0` so it doesn't look the same as all the other `select` functions here.

Reply:
It's not mutating a reference to the argument. It takes ownership over it and mutates it, which is consistent with the other functions (they just happen to embed the owned argument into other things).